pipelines/LSSTComCamSim/cpDefects.yaml

```diff
@@ -1,4 +1,29 @@
-description: cp_pipe DEFECT calibration construction from combined images.
+description: cp_pipe LSSTComCamSim defect calibration construction.
 instrument: lsst.obs.lsst.LsstComCamSim
 imports:
-  - location: $CP_PIPE_DIR/pipelines/_ingredients/cpDefects.yaml
+  - location: $CP_PIPE_DIR/pipelines/_ingredients/cpDefectsLSST.yaml
```

Comment:
Have we ever run this on LSSTComCamSim data?

Reply:
No, actually.  I was asked to leave it in just in case ComCamSim adds defects in the future.  But I actually think that it is more likely we will just use the existing defects and apply them and not "find" them so this may not be necessary.  But wanted to leave the option open.